adapter/src/main/java/com/twilio/kudu/sql/CalciteKuduEnumerable.java

```diff
@@ -58,7 +58,7 @@ public CalciteKuduEnumerable(final BlockingQueue<CalciteScannerMessage<CalciteRo
       @Override
       public boolean moveNext() {
         if (finished) {
-          logger.info("returning finished");
+          logger.debug("returning finished");
```

Comment:
nit: revert to `logger.info`?

Reply:
I see a bunch of these in the logs while running tests which aren't helpful usually.
```
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
Batch completed for a scanner. Getting next batch
```